client/src/main/java/application/Launcher.java

```diff
@@ -1,46 +1,55 @@
-/*
- * Launcher.java
- * Version: 1.0.0
- * Company: CUBIXEL
- *
- * */
-
 package application;
 
 import application.controller.services.MainConnection;
 import application.view.ViewFactory;
+import java.io.IOException;
 import javafx.application.Application;
+import javafx.application.Platform;
 import javafx.stage.Stage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * CLASS DESCRIPTION:
- * This is the main class for the program. It extends the
+ * This is the main entry for the Client side. It extends the
  * JavaFX Application class and the start() method is the
  * main entry point into the JavaFX application.
  *
- * @author CUBIXEL
- *
+ * @author James Gardner
+ * @see    Logger
+ * @see    MainConnection
+ * @see    ViewFactory
  */
 public class Launcher extends Application {
 
+  /* Logger used by Client. Prints to both the console and to a file 'logFile.log' saved
+   * under resources/logs. This should be passed around the program so that all classes
+   * use the same Logger. */
+  private static final Logger log = LoggerFactory.getLogger(Launcher.class);
+
   @Override
   public void start(Stage stage) throws Exception {
 
     /* Creates the connection between the Client and the Server. The program must
-      * have this connection in order to proceed. */
-    // ########### This could do with some error handling ############################
```

Comment:
In the future could stuff like this be done as //TODO - XYZ. It just makes it easier to track